Sources/EdgeNetworkHandlers/EdgeEndpoint.swift

```diff
@@ -42,7 +42,8 @@ struct EdgeEndpoint {
     ///   - optionalDomain: an optional custom domain for the `EdgeEndpoint`. If not set the default domain is used.
     init(requestType: EdgeRequestType,
          environmentType: EdgeEnvironmentType,
-         optionalDomain: String? = nil) {
+         optionalDomain: String? = nil,
+         overwritePath: String? = nil) {
```

Comment:
To keep in line with the naming of the other parameters, maybe rename to `optionalPath` ?